addons/account/models/account_journal.py

```diff
@@ -1094,3 +1114,51 @@ def _process_reference_for_sale_order(self, order_reference):
         '''
         self.ensure_one()
         return order_reference
+
+    # -------------------------------------------------------------------------
+    # E-Invoice Related Methods
+    # -------------------------------------------------------------------------
+
+    def _notify_einvoices_received(self, moves):
+        self.ensure_one()
+
+        if not (einvoice_count := len(moves)):
+            return
+
+        self._send_einvoice_received_mail_if_needed(einvoice_count)
+
+    def _send_einvoice_received_mail_if_needed(self, einvoice_count: int):
+        self.ensure_one()
+        if not all((
+            einvoice_count,
+            self.notify_on_incoming_einvoice,
+            self.incoming_einvoice_notification_email,
+        )):
+            return
+
+        mail_template = self.env.ref('account.mail_template_einvoice_notification', raise_if_not_found=False)
+        if not mail_template:
+            return
+        mail_template.with_context(einvoice_count=einvoice_count).send_mail(self.id, force_send=True)
+
+    @api.depends('type')
+    def _compute_show_fetch_einvoices_button(self):
```

Comment:
```suggestion
    def _compute_show_fetch_einvoices_button(self):
        # TO OVERRIDE
```
Looks like a newbie error if you don't know it's extended in submodules :)
It's imo a good practice to put a short comment to avoid any doubt.
Same below.